### PR TITLE
Add cosminbg pre genesis transactions

### PR DESCRIPTION
**Pull request by cosminbg**
## Discord handler
cosmin_bg

## Checklist before opening a pull request
- [ X ] My submissions follow the [instructions](https://hackmd.io/CNzB-rsNQ1ykRHkj8EDbTA?view)
- [ X ] Im aware my submission will not be modifiable/delatable if merged

### CODE REVIEW

File: genesis/transactions.toml
Comment (Fraccaman) [inline genesis/transactions.toml:1]: you dont have to modify this file, just leave it as it is

File: transactions/cosminbg-bond.toml
Comment (Fraccaman) [inline transactions/cosminbg-bond.toml:1]: this doesn;t look like a bond transaction, but more like a init-validator tx ?

Comment (cosminbg) [reply transactions/cosminbg-bond.toml:1]: unfortunately I noticed that the bond transaction can't be done as we don't have tokens. Pity we worked on the setup. I don't see any workaround this issue.

Comment (Fraccaman) [reply transactions/cosminbg-bond.toml:1]: then remove it

File: transactions/cosminbg-validator.toml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+authorization = "signam1qye2jmatth5lsmtcnee9ap8d5x8ruqr6g30p6rm9h2htyfk0ral6sz4vr0h6saw9usjn5cs5t5rkntaz6ha3h523vw3mlu93nnpetvmaqqnlmer8"
+
+[validator_account.eth_cold_key]
+pk = "tpknam1qype4sw2ay5zs3986ut3m3ykwtgymupfr0mghtfpy9xsvzlh0mvurgqmw29jx"
+authorization = "signam1qxfhw5uc5dj8gwyd0frucns8jvsky4wjdah8rjxxr8yws4yfxjut72xjngxw34c2jqtcqkujzvlwjrkuuy52q4tr4y827aetruq5kfk3qqj9dfty"
+
+[validator_account.metadata]
+email = "“hello@clorophilla.io\nVALIDATOR_NAME=“Clorophilla"
+description = ""
+name = "“Clorophilla # This is optional\nSELF_BOND_AMOUNT=1000000 # Set this to the amount of NAM you want to self-bond. This must be less than or equal to the amount of NAM allocated to the pre-genesis keys in the  that was published by the network coordinator.\nVALIDATOR_ALIAS=“validator"
```
Comment (Fraccaman) [inline transactions/cosminbg-validator.toml:36]: I think you should also check the data here, seems like something is messed up

**Review by Fraccaman — APPROVED** [review]

**Review by cosminbg — COMMENTED** [review]
thanks for the comments! updated

**Review by Fraccaman — DISMISSED** [review]

**Comment by Fraccaman** [comment]
signatures are invalid, you might have to regenerate the transaction

**Comment by cosminbg** [comment]
Hi,

I re-generated the file and changed the contents of the original one. Thanks

**Review by Fraccaman — APPROVED** [review]

**Pull request merged**